p2pclient/streams.go

```diff
@@ -53,17 +53,17 @@ func (c *byteReaderConn) ReadByte() (byte, error) {
 }
 
 func readMsgBytesSafe(r *byteReaderConn) (*bytes.Buffer, error) {
-	len, err := binary.ReadUvarint(r)
+	length, err := binary.ReadUvarint(r)
```

Comment:
probably good not to overwrite builtins 😨 good call here

Reply:
Yea,  I was getting a warning